### DISCUSSION TITLE
I am getting this issue while building the latest code.. Kindly help

### DISCUSSION THREAD

**Discussion by shary998** [florisboard/florisboard#1801] [Q&A]
Build command failed.
Error while executing process C:\Users\Shaheryar\AppData\Local\Android\Sdk\cmake\3.18.1\bin\ninja.exe with arguments {-C D:\Floris New Board\florisboard-master\app\.cxx\Debug\6621703z\arm64-v8a florisboard-native}
ninja: Entering directory `D:\Floris New Board\florisboard-master\app\.cxx\Debug\6621703z\arm64-v8a'

ninja: error: 'D:/Floris New Board/florisboard-master/app/src/main/icu4c/prebuilt/jniLibs/arm64-v8a/libicuuc.a', needed by 'D:/Floris New Board/florisboard-master/app/build/intermediates/cxx/Debug/6621703z/obj/arm64-v8a/libflorisboard-native.so', missing and no known rule to make it

**Comment by patrickgold** [comment]
You need to initialize the git submodules, which contains files required to build this project: 

``git submodule update --init --recursive``

Then it should work.

**Reply by MuhammadZahab** [reply]
I have initialized submodules. I am still having build issues on mac.  Can you please check this post too. https://stackoverflow.com/questions/77329492/error-while-building-florisboard-open-source-on-mac-os-sonoma-ninja-build-stop?noredirect=1#comment136327364_77329492
